pqueue/pqueue.go

```diff
@@ -134,6 +137,10 @@ func (pq *PQueue) storeMessage(msg *PQMessage, payload string) error {
 	pq.allMessagesMap[msg.Id] = msg
 	pq.trackExpiration(msg)
 	pq.availableMsgs.Push(msg.Id, msg.Priority)
+	select {
```

Comment:
Ok, will send the message only if msg was added in empty queue. PopWait will handle this usecase.